tests/alcatel_aos/show_interfaces_status/alcatel_aos_show_interfaces_status_R8.yml

```diff
@@ -0,0 +1,152 @@
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/2"
+  - admin_status: "en"
+    auto_negotiation: "en"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: "Full"
+    detected_fec: "DIS"
+    detected_pause: ""
+    detected_speed: "1000"
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/3"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/4"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/5"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/6"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/7"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "10000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/8"
+  - admin_status: "en"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "20000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
+    detected_pause: ""
+    detected_speed: ""
+    eee: "dis"
+    link_trap: "en"
+    port: "1/1/9"
+  - admin_status: "dis"
+    auto_negotiation: "dis"
+    configured_duplex: "Full"
+    configured_fec: "AUTO"
+    configured_hybrid: ""
+    configured_pause: ""
+    configured_speed: "20000"
+    detect_hybrid: ""
+    detected_duplex: ""
+    detected_fec: ""
```

Comment:
```suggestion
    detected_duplex: "Full"
    detected_fec: "DIS"
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: "1000"
    eee: "dis"
    link_trap: "en"
    port: "1/1/1"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/2"
  - admin_status: "en"
    auto_negotiation: "en"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: "Full"
    detected_fec: "DIS"
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: "1000"
    eee: "dis"
    link_trap: "en"
    port: "1/1/3"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/4"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/5"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/6"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/7"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "10000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/8"
  - admin_status: "en"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "20000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
    detected_pause: ""
    detected_speed: ""
    eee: "dis"
    link_trap: "en"
    port: "1/1/9"
  - admin_status: "dis"
    auto_negotiation: "dis"
    configured_duplex: "Full"
    configured_fec: "AUTO"
    configured_hybrid: ""
    configured_pause: ""
    configured_speed: "20000"
    detected_duplex: ""
    detected_fec: ""
    detected_hybrid: ""
```

Update for DETECTED instead of DETECT similar to other parsed data.